vars/buildDockerAndPublishImage.groovy

```diff
@@ -196,7 +196,7 @@ def call(String imageShortName, Map userConfig=[:]) {
               }
             } finally {
               recordIssues(
-                  skipPublishingChecks: (env.BRANCH_IS_PRIMARY),
+                  skipPublishingChecks: "${env.BRANCH_IS_PRIMARY}",
```

Comment:
```suggestion
                  skipPublishingChecks: env.BRANCH_IS_PRIMARY,
```

There are no need to interpolate here: we are in Groovy so you can directly call the variable. Types are compatible (both are booleans) so you're good